sys-fs/xfsdump/xfsdump-3.1.9-r1.ebuild

```diff
@@ -24,7 +24,6 @@ RDEPEND="
 DEPEND="${RDEPEND}
 	nls? (
 		sys-devel/gettext
-		elibc_uclibc? ( dev-libs/libintl )
```

Comment:
needs to be kept with virtual/libintl instead, although it should really be in RDEPEND too